apps/connect/vite.config.ts

```diff
@@ -61,6 +61,8 @@ export default defineConfig(({ mode }) => {
     nodePolyfills({
       include: ["events"],
       globals: {
+        Buffer: false,
+        global: false,
```

Comment:
Disabling `global` polyfill removes the import error on the `RunAsap` module in `DocumentDrive` which checks if some `node` only globals are available.